reporthandling/helpers/v1/listing.go

```diff
@@ -89,62 +98,63 @@ func (all *AllLists) Append(status apis.ScanningStatus, str ...string) {
 
 	switch status {
 	case apis.StatusPassed:
-		all.passed = append(all.passed, str...)
+		if all.passed == nil {
```

Comment:
you don't need this check. All args are variadic: nil would result in an empty (but not nil) set

Reply:
I need these, otherwise I get:
`panic: assignment to entry in nil map`